xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/job/DeleteRequest.java

```diff
@@ -145,10 +145,12 @@ public void setUpdateLinks(boolean updateLinks)
      * @return {@code true} if the job should update the links that target the old entity reference (before the delete)
      *         from anywhere on the farm, {@code false} if the job should update only the links from the wiki where the
      *         entity was located before the delete
+     * @deprecated not taken into account anymore
      */
+    @Deprecated(since = "14.8RC1")
     public boolean isUpdateLinksOnFarm()
     {
-        return getProperty(UPDATE_LINKS_ON_FARM, false);
```

Comment:
I guess it means the `UPDATE_LINKS_ON_FARM` constant should be deprecated and their usages (notably in velocity template I guess) should be removed no?

Reply:
I deprecated the related APIs, but indeed I forgot the property itself.